templates/terraform/examples/spanner_database_basic.tf.erb

```diff
@@ -0,0 +1,13 @@
+resource "google_spanner_instance" "main" {
```

Comment:
We can delete the test that's identical to this example.

Reply:
Done.